nilearn/conftest.py

```diff
@@ -2,13 +2,12 @@
 import nibabel
 import numpy as np
 import pytest
-from _pytest.doctest import DoctestItem
 from nilearn import image
 
 # we need to import these fixtures even if not used in this module
 from nilearn.datasets._testing import request_mocker  # noqa: F401
 from nilearn.datasets._testing import temp_nilearn_data_dir  # noqa: F401
-from nilearn.version import _compare_version
+from nilearn.version import _compare_version  # noqa: F401
```

Comment:
Why didn't you remove the import ?

Reply:
I think I got confused by the comment above that those imports were needed.
Removed it.
Let's see what CI says.